### DISCUSSION TITLE
Kafka-versions.yaml 0.32.0

### DISCUSSION THREAD

**Discussion by chandans21j** [strimzi/strimzi-kafka-operator#8164] [General]
Can we use credentials in kafka-versions.yaml file? if yes then how ?

**Comment by vishalsingh9990** [comment]
No we cannot use it

**Comment by CraigDouviaP** [comment]
you can use credentials in the kafka-versions.yaml file by specifying them in the sasl section.

**Reply by scholzj** [reply]
There is no SASL section in the kafka-versions.yaml file. You might be able to specify some HTTP Basic Auth credentials as part of the URL. But nothing more. Assuming this is the file we are talking about: https://github.com/strimzi/strimzi-kafka-operator/blob/main/kafka-versions.yaml
